MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/Utilities.java

```diff
@@ -0,0 +1,33 @@
+package com.mapbox.mapboxandroiddemo;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.DrawableRes;
+import android.support.annotation.NonNull;
+import android.support.v4.content.res.ResourcesCompat;
+
+import com.mapbox.mapboxsdk.annotations.Icon;
+import com.mapbox.mapboxsdk.annotations.IconFactory;
+
+public class Utilities {
```

Comment:
Can we call this class simply `Utils` to keep consistency with our other SDKs and Java projects?